src/helpers/constants/validation.schemas.js

```diff
@@ -56,11 +56,21 @@ export default {
       .required(),
     profileImage: Joi.string(),
     facebook: Joi.string(),
-    twitter: Joi.string(),
+    twitter: Joi.string()
   }),
   rateArticle: Joi.object().keys({
     rating: number
       .valid(numbers.ONE, numbers.TWO, numbers.THREE, numbers.FOUR, numbers.FIVE)
       .required()
   }),
+  reportArticle: Joi.object().keys({
+    reason: number.required()
```

Comment:
can I report even with `0` ?

Reply:
nah, you can't cause you report what's in the database? and it saves it counting from 1 not 0

Reply:
I would suggest you validate that @MCFrank16

Reply:
yeah lemme work on it